packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -300,7 +301,11 @@ const withChunking = (config) => {
                             if (
                                 EXT_EXTENDS &&
                                 EXT_OVERRIDES_DIR &&
-                                module?.context?.includes(`/${EXT_EXTENDS}/`)
+                                module?.context?.includes(
+                                    `${path.sep}${
+                                        path.sep === '/' ? EXT_EXTENDS : EXT_EXTENDS_WIN
```

Comment:
Non blocking

i feel like there has to be a way to make the code platform independent, maybe `path.normalize`(?)

Condition like this (`if (windows)`) increase the maintenance cost because the code path is different for mac and windows and forces us to test on different platforms.

That being said, this is a non blocking comment, hopefully we can revisit this later.

Reply:
ideally, all variables should be sanitized by `path.resolve`, we should not use the values that came out directly from string replacement / concatenation.

Reply:
I'm open to that, I think I've got enough on my plate that I'm not thinking about it, but I **very much** welcome other folks to embark on the process of becoming subject matter experts in this domain and correcting things like this 🏅 

Meanwhile, I gotta go write docs